src/main/resources/record_3.0/work-3.0.xsd

```diff
@@ -197,8 +197,7 @@
 		<xs:sequence>
 			<xs:element name="contributor-sequence" type="work:contributor-sequence"
 				minOccurs="0" />
-			<xs:element name="contributor-role" type="common:contributor-role"
-				minOccurs="0" />
+			<xs:element name="contributor-role" type="xs:string" minOccurs="0" />
```

Comment:
@bobcaprice we don't want to modify the XSD's since this could potentially break members that have been using this XSD to build their integrations.

@wjrsimpson @TomDemeranville not sure if we should proceed with this card, since all possible scenarios will require us to update the works XSD and probably the Fundings XSD as well.

Should we leave this for 3.1?

Reply:
@bobcaprice the [contributor-role](https://github.com/ORCID/orcid-model/blob/master/src/main/resources/common_3.0/common-3.0.xsd#L905) is already an string in the common xsd for V3.0, so, this change is unnecessary, could you please roll that back and test it again?

Thanks